src/home_robot_hw/home_robot_hw/env/stretch_abstract_env.py

```diff
@@ -230,6 +275,11 @@ def _create_pubs_subs(self):
             )
             sys.exit()
 
+        self.ros_joint_names = []
```

Comment:
why is ros_joint_names not in constants? Is there any specific reason?

Reply:
discussed offline